data/stoa0089/stoa001/__cts__.xml

```diff
@@ -3,12 +3,12 @@
            
   <ti:edition urn="urn:cts:latinLit:stoa0089.stoa001.perseus-lat2" workUrn="urn:cts:latinLit:stoa0089.stoa001" xml:lang="lat">
               <ti:label xml:lang="lat">Carminum minorum corpusculum</ti:label>
-              <ti:description xml:lang="eng">Claudian, with an English translation, in Two Volumes, Vol. II. Platnauer, Maurice, editor. Harvard University Press: Cambridge, MA, London, England. 1922 (printing).</ti:description>
+              <ti:description xml:lang="eng">Claudian. Claudian, with an English translation, Volume 2. Platnauer, Maurice, editor. Harvard University Press: Cambridge, MA, London, England. 1922 (printing).</ti:description>
             </ti:edition>
```

Comment:
I don't know why `(printing)` is included here — 1922 is already public domain.

Also, should be a comma before 1922 instead of period.

Reply:
I will fix the period and I'm not sure why the (printing) is there. I was thinking of taking it out but then didn't.